ui/src/components/HydraRawRdfForm.vue

```diff
@@ -52,6 +53,7 @@ export default class HydraRawRdfForm extends Vue {
 
   parseError: string | null = null
   editorQuads: Quad[] | null = null
+  editorPrefixes = ['hydra', 'rdf', 'rdfs', 'schema', 'xsd']
```

Comment:
FYI `rdf(s)` and `xsd` are not necessary. Always added anyway

Reply:
"Explicit is better than implicit"™ 😉